tools/render-test/cpu-memory-binding.cpp

```diff
@@ -158,17 +216,42 @@ slang::VariableLayoutReflection* CPUMemoryBinding::getParameterByName(const char
             return parameter;
         }
     }
+    
     return nullptr;
 }
 
+slang::VariableLayoutReflection* CPUMemoryBinding::getEntryPointParameterByName(const char* name)
+{
+    const int parameterCount = int(m_entryPoint->getParameterCount());
+    for (int i = 0; i < parameterCount; ++i)
+    {
+        auto parameter = m_entryPoint->getParameterByIndex(i);
+        // If has a semantic we will ignore
+        if (parameter->getSemanticName())
```

Comment:
I don't think this is a robust way to skip "varying" parameters, because the user can do stuff like:

```
struct V2F { float3 normal : NORMAL; ... }; // semantics on fields
float4 main( V2F v2f ) // no semantic on parameter
```

In principle the Right Way to do an "is varying" check would be to look at the resource kinds consumed by a parameter (with the caveat that we currently have system input/outputs consume *no* resources of any kind...). We should probably have a Slang reflection API function that can determine if a var layout represents a varying parameter, so that users don't have to try to write it by hand.